test/integration/webhook/core/tas_test.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package core
+
+import (
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	kueuealpha "sigs.k8s.io/kueue/apis/kueue/v1alpha1"
+	"sigs.k8s.io/kueue/pkg/util/testing"
+)
+
+var _ = ginkgo.Describe("Topology validations", func() {
```

Comment:
We don't have a webhook for topology. The validations are don't by CRD schema so far, so I would suggest moving the tests to  `test/integration/tas`

Reply:
I see, I think keeping them in webhook folder is misleading. I see that many validation asserts are in the package, but I think this is mostly for historical reasons when almost all validation was done in webhooks. As we move gradually the validations out of webhooks, we can also move the testing out.

Unless, we rename the folder as test/integration/validation/core`, but I think it is better to keep them feature per feature in the long run.

Reply:
> Unless, we rename the folder as test/integration/validation/core`, but I think it is better to keep them feature per feature in the long run.

The current `/test/integration/webhook/core` package has mutating and validating testings. So I think that we can rename it `test/integration/validation/core`. For now, let's move tas validation testings to tas package.

Reply:
Done.